store/src/main/java/com/dropbox/android/external/store4/StoreBuilder.kt

```diff
@@ -36,6 +36,8 @@ interface StoreBuilder<Key, Output> {
     fun scope(scope: CoroutineScope): StoreBuilder<Key, Output>
     fun cachePolicy(memoryPolicy: MemoryPolicy?): StoreBuilder<Key, Output>
     fun disableCache(): StoreBuilder<Key, Output>
+    fun fetchBufferSize(fetchBufferSize: Int): StoreBuilder<Key, Output>
+    fun keepFetcherAlive(keepFetcherAlive: Boolean): StoreBuilder<Key, Output>
```

Comment:
also, i think we do not want to expose buffer here do we?
come to think of it, we need to figure out how this can be used by store. imo, this is a weird case as wanted results from previously fetched data seems very unusual for store and it kind of only makes sense if there is no source of truth.

Reply:
I don't have a strong opinion on the API, we can expose it only up the multicaster for now until we decide how to expose it via store